lib/rubocop/cop/style/symbol_proc.rb

```diff
@@ -42,6 +42,13 @@ module Style
       #   # good
       #   something.map(&:upcase)
       #
+      #   # good  - if there are comment in either position
```

Comment:
Maybe this is a corner case and it would be better to provide `AllowComments: false (default)` option. if `AllowComments: false`, it will not be auto-corrected. And it would be user-friendly to mention it in the document.

Reply:
@koic It's certainly better that way. Thank you so much.
I've made the correction, could you please review again?